subprojects/docs/src/docs/userguide/authoring-builds/test_kit.adoc

```diff
@@ -80,6 +80,14 @@ It is a common practice to implement any custom build logic (like plugins and ta
 
 The GradleRunner uses the <<third_party_integration.adoc#embedding,Tooling API>> to execute builds. An implication of this is that the builds are executed in a separate process (i.e. not the same process executing the tests). Therefore, the test build does not share the same classpath or classloaders as the test process and the code under test is not implicitly available to the test build.
 
+[NOTE]
+====
+GradleRunner supports the same range of Gradle versions as the Tooling API.
+At the time of writing Gradle 3.0 is the lowest supported version.
```

Comment:
🤔 I'm afraid this version number will rot. I think we should only link to the compatibility matrix where that content is maintained.

Reply:
That's a valid point, here's my fix proposal: https://github.com/gradle/gradle/pull/26331